app/Controllers/feedController.php

```diff
@@ -891,7 +909,7 @@ public function reloadAction(): void {
 		}
 
 		//Re-fetch articles as if the feed was new.
-		$feedDAO->updateFeed($feed->id(), [ 'lastUpdate' => 0 ]);
```

Comment:
formatting changes only; unfortunately these "escaped" my checks and were committed